src/main/java/org/commcare/xml/CaseXmlParser.java

```diff
@@ -204,6 +204,18 @@ private void updateCase(Case caseForBlock, String caseId) throws InvalidStructur
                 case "external_id":
                     caseForBlock.setExternalId(value);
                     break;
+                case "category":
```

Comment:
(non-actionable) I have mixed feelings about keeping these here (since they aren't doing anything other than the core code path), but it makes sense to stay consistent with the `external_id` behavior.

Reply:
I felt the same way. I am confused why those `case`s exist since the data is saved the same as the `default`